src/selectors/breach.js

```diff
@@ -11,19 +11,17 @@ export const selectIsBreachModalOpen = ({ breach }) => {
 };
 
 export const selectBreachModalTitle = ({ breach }) => {
-  const { forBatch, forFridge } = breach;
+  const { forItem, forBatch, forFridge } = breach;
 
-  if (!forBatch && !forFridge) return '';
+  if (!forBatch && !forFridge && !forItem) return '';
 
-  if (forBatch) {
-    const { batch } = breach;
-    const { itemName } = batch;
+  if (forBatch || forItem) {
+    const { itemName } = breach;
 
     return `${vaccineStrings.temperature_breaches_for} ${itemName}`;
   }
 
-  const { fridge } = breach;
-  const { description } = fridge;
+  const { fridgeName } = breach;
```

Comment:
I should just make the store have a `name` and then there can reduce a lot of this by a lot, I think